src/mgtransport.h

```diff
@@ -19,9 +19,22 @@
 extern "C" {
 #endif
 
+#ifndef __EMSCRIPTEN__
+
 #include <openssl/bio.h>
 #include <openssl/err.h>
 #include <openssl/ssl.h>
+
+#else
+// This is a small hack to allow the current code to compile without openSSL
+// support. This is guranteed to work because the API is defined natively by the
+// implementation language that consumes the mgclient wasm module.
```

Comment:
What about pushing this to compilation time? I mean everything that is SSL related, should be excluded from WASM build. In that case people cannot even specify SSL mode for connection.

Reply:
I think the whole secure transport and its functions can be ifdef-ed out, but it shouldn't be a big issue in my opinion.